Updated release notes for 28th Nov (#387)
cyclr/docs: pages/embedding/release-notes.md

## 2022

### 28th November 2022
- Performance: Improved Cycle execution performance for Cycles with lots of decision steps

### 21st November 2022
- Performance: Reduced latency for synchronous webhooks
- Script: Added HMAC-SHA512 to cyclr_sign

### 14th November 2022

> - API: Can get and set Data Retention values for Accounts

Reviewer: ```suggestion
- API: Added the ability to get and set Data Retention values for accounts
```

> - Script: script_parameters now accessible in ``before_oauth2_authorise``,``before_oauth2_token``, ``after_oauth2_token``, ``before_oauth2_refresh`` &amp; ``after_oauth2_refresh``

Reviewer: ```suggestion
- Script: Made `script_parameters` accessible in ``before_oauth2_authorise``,``before_oauth2_token``, ``after_oauth2_token``, ``before_oauth2_refresh`` &amp; ``after_oauth2_refresh``
```

### 10th October 2022

> - API: Marketplace Noticiations now include Marketplace tags

Reviewer: ```suggestion
- API: Included Marketplace tags in the Marketplace notifications.
```

Reviewer: spelling error
